### DISCUSSION TITLE
New app: Flipper Authenticator (TOTP)

### DISCUSSION THREAD

**Discussion by akopachov** [djsime1/awesome-flipperzero#72] [Ideas]
Would appreciate adding [Flipper Authenticator](https://github.com/akopachov/flipper-zero_authenticator/) app to the list of apps.

**Comment by SnayDev** [comment]
amazing idea. Thank you

**Comment by djsime1** [comment]
Added in update 70.

**Reply by akopachov** [reply]
Thank you
